lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context_and_proc.h

```diff
@@ -339,15 +339,13 @@ typedef struct amf_context_s {
 } amf_context_t;
 
 // Amf-Map Declarations:
-// Map- Key: uint64_t , Data: uint64_t
-typedef magma::map_s<uint64_t, uint64_t> map_uint64_uint64_t;
 // Map Key: guti_m5_t Data: uint64_t;
 typedef magma::map_s<guti_m5_t, uint64_t> map_guti_m5_uint64_t;
 
 typedef struct amf_ue_context_s {
-  map_uint64_uint64_t imsi_amf_ue_id_htbl;    // data is amf_ue_ngap_id_t
-  map_uint64_uint64_t tun11_ue_context_htbl;  // data is amf_ue_ngap_id_t
-  map_uint64_uint64_t
+  magma::map_uint64_uint64_t imsi_amf_ue_id_htbl;    // data is amf_ue_ngap_id_t
```

Comment:
Since we will use protobuf generated maps instead to hold these hashtables, as explained on my previous comment, we should define these with `google::protobuf::Map<uint64_t, uint64_t>` signature instead: https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.map